bson/mgocompat/registry.go

```diff
@@ -31,17 +31,17 @@ var (
 	tSetter         = reflect.TypeOf((*Setter)(nil)).Elem()
 )
 
-// mgoRegistry is the mgo compatible bsoncodec.Registry. It contains the default and
+// MgoRegistry is the mgo compatible bsoncodec.Registry. It contains the default and
 // primitive codecs with mgo compatible options.
-var mgoRegistry = newMgoRegistryBuilder().Build()
+var MgoRegistry = NewMgoRegistryBuilder().Build()
```

Comment:
I would call this just Registry. We are already in the `mgocompat` package, so I think `mgocompat.Registry` is clear enough.

Same with the naming below. No need to repeat `mgo` everything I don't think.

Reply:
done